### PR TITLE
Fix issue when devices are part of inventory and not managed by CloudVision

### PR DESCRIPTION
**Pull request by carlbuchmann**
Port PR 171 to releases/v1.0.x

### CODE REVIEW

**Comment by titom73** [comment]
Seems a change is missing in this PR since output is not the same for a playbook using __`devel`__ and __`PR173`__

Investigating why we have such behavior

## Playbook configuration

```yaml
- name: Configuration deployment with CVP
  hosts: cv_server
  connection: local
  gather_facts: false
  collections:
    - arista.avd
    - arista.cvp
  vars:
    run_tasks: false
  tasks:
    - name: run CVP provisioning
      import_role:
        name: eos_config_deploy_cvp
      vars:
        container_root: 'DC1_FABRIC'
        configlets_prefix: 'AVD'
        device_filter: 'DC1-LEAF'
        execute_tasks: false
        state: present
```

## Configlet structure from PR

```yaml
CVP_CONFIGLETS: {}
CVP_TOPOLOGY:
```

## Configlet structure output from devel branch

```yaml
CVP_CONFIGLETS:
  AVD_DC1-LEAF1A: "..."
  AVD_DC1-LEAF1B: "..."
  AVD_DC1-LEAF2A: "..."
  AVD_DC1-LEAF2B: "..."
CVP_TOPOLOGY:
```

**Comment by carlbuchmann** [comment]
@titom73 also cherry-picked commits from PR #136, to resolve the issue you encountered.

File: ansible_collections/arista/avd/roles/eos_config_deploy_cvp/tasks/main.yml [hunk 1]
```diff
@@ -7,7 +7,8 @@
     container_root: '{{ container_root }}'
     configlet_dir: '{{playbook_dir}}/intended/configs'
```
Comment (titom73) [inline ansible_collections/arista/avd/roles/eos_config_deploy_cvp/tasks/main.yml]: Should be updated with cherry-picking commit a788b70ebf6bef282146bbe1b0f43941b6a93066

**Review by titom73 — CHANGES_REQUESTED** [review]
Should be updated with cherry-picking commit a788b70ebf6bef282146bbe1b0f43941b6a93066

File: ansible_collections/arista/avd/roles/eos_config_deploy_cvp/tasks/main.yml (hunk 1 above)
Comment (carlbuchmann) [reply ansible_collections/arista/avd/roles/eos_config_deploy_cvp/tasks/main.yml]: done

**Review by titom73 — APPROVED** [review]
LGTM

**Pull request merged**